cfgov/unprocessed/css/molecules/autocomplete.less

```diff
@@ -0,0 +1,39 @@
+
+.m-autocomplete {
+    position: relative;
+}
+
+.m-autocomplete_results {
+    box-sizing: border-box;
+    width: 100%;
+    max-height: 200px;
```

Comment:
Added in https://github.com/cfpb/cfgov-refresh/pull/2985/commits/684ce026628936b3482a283f7e72bcb69537d154